docs/index.md

```diff
@@ -49,16 +52,24 @@ containers on the same host. For details, see
 
 ## Pre-requisites for running Swarm
 
-You must install Docker 1.4.0 or later on all nodes. While each node's IP need not
-be public, the Swarm manager must be able to access each node across the network.
+Each machine of your cluster must be running Docker 1.4.0 or later.
 
-To enable communication between the Swarm manager and the Swarm node agent on
-each node, each node must listen to the same network interface (TCP port).
-Follow the set up below to ensure you configure your nodes correctly for this
-behavior.
+By default, Docker listens on `unix:///var/run/docker.sock` to allow only local connections.
 
-> **Note**: Swarm is currently in beta, so things are likely to change. We
-> don't recommend you use it in production yet.
+To enable communication between Swarm and Docker running on different machines,
+each Docker must be 
+[configured] (https://docs.docker.com/articles/basics/#bind-docker-to-another-hostport-or-a-unix-socket)
+to listen to a TCP port.
+
+While each machine's IP need not be public, the Swarm manager must be able to access
+each machine across the network.
+
+Follow the set up below to configure your machines correctly for this behavior.
+
+> Warning: Changing the default `docker` daemon binding to a TCP port will
+> increase your security risks by allowing anyone with access to that port
+> full Docker access; so it is not advisable on an open network.
```

Comment:
Might want to add that having access to the API is equivalent to having root access to the host. I think there's a note on that somewhere in the docker docs.

Perhaps add a link to https://docs.docker.com/articles/security/#docker-daemon-attack-surface